linux_sql/sql/queries.sql

```diff
@@ -23,14 +23,15 @@ select
 	avg
 		( host_info.total_mem - host_usage.memory_free)/host_info.total_mem) * 100
 ) as used_memory percentage
-FROM host_usage
+
+FROM 
+	host_usage
```

Comment:
Remove extra newlines. Use https://codebeautify.org/sqlformatter to see what it can look like. Otherwise it is good to merge when you want.

Reply:
Hi. Removed all extra newlines.